### PR TITLE
Veoss 796

### PR DESCRIPTION
**Pull request by oliwiaKot**

### CODE REVIEW

File: test/uk/gov/hmrc/onestopshopforexrates/connectors/DesConnectorSpec.scala [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+        result mustBe Left(expectedResponse)
+      }
+    }
+
+    "must return Conflict Found when data is already present in core" in {
+
+      val url = "/one-stop-shop-returns-stub/oss/referencedata/v1/exchangerate"
+
+      val errorResponseJson = """{}"""
```
Comment (DevAndyroo) [inline test/uk/gov/hmrc/onestopshopforexrates/connectors/DesConnectorSpec.scala:142]: Can we add a test where we have an example error response and parse it correctly?

File: app/uk/gov/hmrc/onestopshopforexrates/config/ModuleBindings.scala [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import uk.gov.hmrc.onestopshopforexrates.scheduler.jobs.{RetrieveAndSendForexDataJob, RetrieveAndSendForexDataJobImpl}
+import uk.gov.hmrc.onestopshopforexrates.services.{ExchangeRatesService, ExchangeRatesServiceImpl}
+
+import java.time.{Clock, ZoneOffset}
+
+class ModuleBindings extends Module {
+
+  override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] = Seq(
+    bind(classOf[Clock]).toInstance(Clock.systemDefaultZone.withZone(ZoneOffset.UTC)),
+    bind[ExchangeRatesService].to[ExchangeRatesServiceImpl].eagerly(),
```
Comment (DevAndyroo) [inline app/uk/gov/hmrc/onestopshopforexrates/config/ModuleBindings.scala]: Why do we need to eagerly inject this?

**Review by DevAndyroo — CHANGES_REQUESTED** [review]

File: app/uk/gov/hmrc/onestopshopforexrates/config/ModuleBindings.scala (hunk 2 above)
Comment (oliwiaKot) [reply app/uk/gov/hmrc/onestopshopforexrates/config/ModuleBindings.scala]: We actually didn't need eager binding of the service, just the job itself, thanks for spotting this! Fixed now

File: test/uk/gov/hmrc/onestopshopforexrates/connectors/DesConnectorSpec.scala (hunk 1 above)
Comment (oliwiaKot) [reply test/uk/gov/hmrc/onestopshopforexrates/connectors/DesConnectorSpec.scala:142]: done now, we changed one of the specs to do that

**Review by DevAndyroo — APPROVED** [review]

**Pull request merged**